tensorflow/python/keras/layers/core.py

```diff
@@ -651,13 +651,16 @@ class Lambda(Layer):
   The `Lambda` layer exists so that arbitrary TensorFlow functions
   can be used when constructing `Sequential` and Functional API
   models. `Lambda` layers are best suited for simple operations or
-  quick experimentation. For more advanced use cases, subclassing
-  `keras.layers.Layer` is preferred. One reason for this is that
-  when saving a Model, `Lambda` layers are saved by serializing the
-  Python bytecode, whereas subclassed Layers are saved via overriding
-  their `get_config` method and are thus more portable. Models that rely
-  on subclassed Layers are also often easier to visualize and reason
-  about.
+  quick experimentation. For more advanced usecases, follow 
+  [this guide](https://www.tensorflow.org/alpha/guide/keras/custom_layers_and_models) 
+  for subclassing `tf.keras.layers.Layer`. 
+  
+  The main reason to subclass `tf.keras.layers.Layer` instead using a 
+  `Lambda` layer is for saving and inspecting a Model. `Lambda` layers 
+  are saved by serializing the Python bytecode, whereas subclassed 
+  Layers are saved via overriding their `get_config` method. Overriding 
+  `get_config` improves portability of Models and are easier to inspect,
+  visualize and reason about.
```

Comment:
I think that we can keep the existing text instead of this para "One reason for this is that
  when saving a Model, `Lambda` layers are saved by serializing the
  Python bytecode, whereas subclassed Layers are saved via overriding	  
  their `get_config` method and are thus more portable. Models that rely
  on subclassed Layers are also often easier to visualize and reason
  about."

Reply:
I agree that we should provide a link. What i meant was that instead of the second para after the link "The main reason to subclass .... visualize and reason about." we can keep the old paragraph.

The sentence  "The main reason to subclass `tf.keras.layers.Layer` instead using a `Lambda` layer is for saving and inspecting a Model." is not grammatically correct.

Reply:
You are right. I'll try to fix the sentence, to provide an alternative to the original sentence still.